examples/upgradeable-contracts/set-code-hash/new-increamenter/lib.rs

```diff
@@ -0,0 +1,46 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+use ink_lang as ink;
+
+#[ink::contract]
+pub mod incrementer {
+
+    #[ink(storage)]
+    pub struct Incrementer {
+        count: u32,
+    }
+
+    impl Incrementer {
+        /// Creates a new counter smart contract initialized with the given base value.
```

Comment:
```suggestion
        /// Creates a new counter smart contract initialized with the given base value.
        ///
        /// Note that with our upgrade-workflow this constructor will never actually be called,
        /// since we merely replace the code used to execute a contract that was already
        /// initiated on-chain.
```